src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestChannel.java

```diff
@@ -244,6 +258,9 @@ public interface SnowflakeStreamingIngestChannel {
    * @param offsetToken offset of given row, used for replay in case of failures. It could be null
    *     if you don't plan on replaying or can't replay
    * @return insert response that possibly contains errors because of insertion failures
+   * @throws net.snowflake.ingest.utils.SFException if the channel is not considered "valid".
```

Comment:
nit: should we also @link the exception here?

Reply:
Good question - I don't think that's needed when its the first value supplied to `throws` or say `return`?

Reply:
Will leave as-is for now.